README.md

```diff
@@ -1,23 +1,176 @@
 # ModelDB: A system to manage ML models
 **Website**: [http://modeldb.csail.mit.edu](http://modeldb.csail.mit.edu)
 
+**See the ModelDB frontend in action**: [Watch the video](https://youtu.be/m2PhgEkx8-E)
```

Comment:
I don't think it's possible to embed videos. I can only make a preview image appear which when clicked will still lead users to Youtube and not play the video within the readme.